lib/chef/knife/winrm_base.rb

```diff
@@ -96,6 +96,10 @@ def self.included(includer)
             :default => :verify_peer,
             :proc => Proc.new { |verify_mode| verify_mode.to_sym }
 
+          option :ssl_peer_fingerprint,
```

Comment:
make sure to add `:ssl_peer_fingerprint` to https://github.com/chef/knife-windows/blob/master/spec/unit/knife/bootstrap_options_spec.rb#L104 which should get most of the tests here green